vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -451,6 +451,7 @@ type VolumeCreateRequest struct {
 type NetworkResource struct {
 	Name       string                      // Name is the requested name of the network
 	ID         string                      `json:"Id"` // ID uniquely identifies a network on a single machine
+	Created    string                      // String representation of the creation timestamp
```

Comment:
This should be a machine-parsable format. int64 is ok, I'd prefer time.Time generally.
Having the created as a string at the API level was just bad.